src/runtime/pkg/resourcecontrol/utils_linux.go

```diff
@@ -21,9 +21,9 @@ import (
 // DefaultResourceControllerID runtime-determined location in the cgroups hierarchy.
 const DefaultResourceControllerID = "/vc"
 
-// ValidCgroupPathV1 returns a valid cgroup path for cgroup v1.
+// ValidCgroupPath returns a valid cgroup path.
 // see https://github.com/opencontainers/runtime-spec/blob/master/config-linux.md#cgroups-path
-func ValidCgroupPathV1(path string, systemdCgroup bool) (string, error) {
+func ValidCgroupPath(path string, systemdCgroup bool) (string, error) {
 	if IsSystemdCgroup(path) {
 		return path, nil
```

Comment:
For CgroupsV2 this previously always performed a transformation:
```
return filepath.Join("/", path), nil
```
Why has this been removed? I would prefer if the commit description or PR description explained this.

Reply:
Moved to #5153 to resolve the issue.